src/applications/disability-benefits/all-claims/tests/schema/maximal-test.json

```diff
@@ -159,7 +163,7 @@
       {
         "cause": "SECONDARY",
         "view:secondaryFollowUp": {
-          "causedByDisability": "Asthma",
+          "causedByDisability": "Diabetes mellitus0",
```

Comment:
Asthma isn't a rated disability so we shouldn't use it as a `causedByDisability`